docs/Course Wiki/ECE Course Offerings/ECE340.md

```diff
@@ -0,0 +1,61 @@
+- [ECE329](ECE329.md) (Corequisite)
+
+Officially [PHYS214](../PHYS%20Course%20Offerings/PHYS214.md) is listed as a prerequisite, while [ECE329](ECE329.md) is listed as a pre-/co-requisite. ECE340 involves many physics concepts such as diffusion and particle distribution, so having completed [PHYS213](../PHYS%20Course%20Offerings/PHYS213.md) is almost as important as PHYS214. In fact, several PHYS213 lectures cover semiconductor physics at a basic level. The first part of ECE340 will involve various topics from PHYS214 such as the concept of discrete energy levels, and a brief discussion of Schrödinger's Equation and its consequences in semiconductors. Relevant topics from ECE329 include Maxwell's equations in material media and carrier mobility. They apply in ECE340 to topics such as p-n junctions and carrier flow, but should be covered early enough in the semester if taken concurrently with ECE 340. Overall, these courses are most helpful in developing an intuition for what is happening at the physics level inside electronic devices.
+
+## When to Take it
+
+Taking this course in accordance with the stated prerequisites and co-requisites is especially important for ECE 340, due to its emphasis on conceptual understanding. Because of the overlap with physics concepts, it is best to take this as soon as possible after the prerequisites. This course is a gateway to many other courses in electrical engineering such as electronic circuits and IC fabrication, and in physics. It is beneficial for students considering careers in anything to do with physical and quantum electronics and circuits to take ECE 340 as early as possible.
+
+## Course Structure
+
+The course is fairly standard with a combination of 11 homeworks, 2 midterms, and a final comprising to total grade. 
+
+Homework assignments are assigned each week and usually consist of three or four problems that may involve drawing band diagrams, derivations, calculations, conceptual explanations, or using an online semiconductor simulation applet. There will be multiple times where certain assumption must be made (temperature independence, neglecting a small concentration, etc) to make ceratin problems solable in a reasonable amount of time. This can be frustrating for students used to solving problems with the exact parameters given to them, but making and justifying these assumptions are part of the course.
+
+Midterm exams for ECE340 are quite imfamous among the 300 level ECE courses. While slight logistical things can vary between semesters (like an equation sheet), in general the tests are 3 questions with several subpart to be solved in 1 hour without a calculator. They are usually even more of a test of time management than ECE210 exams. The strangest quirk about this course is the lack of access to past exams. The testing format also means that there is little similarity to homework problems. This structure makes understanding the concepts taught in ECE340 paramount to success.
```

Comment:
several "subparts" to be solved in 1 hour (add s after subpart)
Can you add what the different logistical things surrounding the cheatsheet are? Would be good to know.